.gitlab-ci.yml

```diff
@@ -1073,8 +1075,6 @@ workflow:
         - test/new-e2e/pkg/**/*
         - test/new-e2e/go.mod
       compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
-  - <<: *if_mergequeue
-    when: never
   - changes:
```

Comment:
maybe for later, as we have 2 `changes` group with the same `compare_to` we can maybe merge them

Reply:
They could be merged, but I think it is clearer like that, we have a separation between modifications made to the framework itself, and modification to Go code impacting the test.